tigerpath/majors_and_certificates/majors/NEU_2018.json

```diff
@@ -0,0 +1,382 @@
+                                    "completed_by_semester": 8,
+                                    "course_list": [
+                                        "MOL 214",
+                                        "MOL 215"
+                                    ]
+                                },
+                                {
+                                    "name": null,
+                                    "max_counted": 1,
+                                    "min_needed": "ALL",
+                                    "explanation": "",
+                                    "double_counting_allowed": true,
+                                    "completed_by_semester": 8,
+                                    "course_list": [
+                                        "ISC 231",
+                                        "ISC 232",
+                                        "ISC 233",
+                                        "ISC 234"
+                                    ]
+                                }
+                            ]
+                        },
+                        {
+                            "name": null,
+                            "max_counted": 1,
+                            "min_needed": null,
+                            "explanation": "",
+                            "double_counting_allowed": false,
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "ISC 235",
+                                "MOL 342"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name": "Behavior",
+                    "max_counted": 1,
+                    "min_needed": 1,
+                    "explanation": "",
+                    "double_counting_allowed": false,
+                    "completed_by_semester": 8,
+                    "course_list": [
+                        "PSY 207: Psychopathology",
+                        "PSY 255: Cognitive Psychology",
+                        "PSY 252: Social Psychology",
+                        "PSY/NEU 345: Sensation & Perception*",
+                        "PSY 254: Developmental Psychology",
+                        "PSY/NEU 338: Animal Learning & Decision Making*",
+                        "EEB 311: Animal Behavior",
+                        "EEB 313: Behavioral Ecology",
+                        "EEB 329: Sensory Ecology"
+                    ]
+                },
+                {
+                    "name": "Quantitative Thinking",
+                    "max_counted": 1,
+                    "min_needed": 1,
+                    "explanation": "",
+                    "double_counting_allowed": false,
+                    "completed_by_semester": 8,
+                    "req_list": [
+                        {
+                            "name": null,

```

Comment:
This requirement is now no longer needed, right?
Can't it just be merged with its parent?